app/models/virtual_cap_pool.rb

```diff
@@ -45,7 +45,6 @@ def has_school?(school)
   def school_can_be_added_to_pool?(school)
     school.responsible_body_id == responsible_body_id &&
       school&.preorder_information&.responsible_body_will_order_devices? &&
-      (school.can_order? || school.can_order_for_specific_circumstances?) &&
```

Comment:
@tonyheadford I think you'll want to relax the eligibility in `SchoolOrderStateAndCapUpdateService` as well.

Reply:
Good spot, thanks!